cves/kernel/CVE-2020-14416.yml

```diff
@@ -393,8 +428,10 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
```

Comment:
Is the suggestion here to just synchronize the resources? I do think this is the correct lesson, but I would like to see a more specific example of how it applies